src/url_aggregator.cpp

```diff
@@ -107,22 +183,46 @@ inline void url_aggregator::copy_scheme(const url_aggregator& u) noexcept {
   ADA_ASSERT_TRUE(validate());
 }
 
+inline void url_aggregator::set_scheme_fast(std::string_view new_scheme_with_colon) noexcept {
```

Comment:
Now I see that, this is a function for having an input with a colon. I think the function name should reflect the functionality.

Reply:
Done.